### PR TITLE
Unify resource loading and `.json` annotation correction

### PR DESCRIPTION
**Pull request by ZZZank**
KubeJS itself is capable of loading resources and datapacks, but there's a load-my-resources mod for res loading, a global-datapack for data loading. That's strange. 
This PR merges resources from `kubejs/asstets/` and `resources/`, and I manually merges the contents of those `.json` with duplicated name. Then, load-my-resources is removed, assets are loaded only through KubeJS. 
I manualy check those changes to make sure nothing gets lost, but additional reviewing is still needed. 

I remembered that there are feedbacks saying that KubeJS cannot properly load some models, I think that's fixed now, because of #125

### CODE REVIEW

**Comment by Laskyyy** [comment]
The way we do datapacks right now is partly due to kubejs historically having issues with datapack priority back in the very oldest versions of astral. Whether this has changed now is unknown, as touching old things and possibly breaking them is something I tend to avoid :P

The load my resources was also a relic of day 1 environments before I discovered kube, but now we just use it as a way to separate language files from textures to help the translators, in fact I think a recent commit went the opposite direction, and I moved every single texture loaded through kube back over to load my resources to see if it amends the texture loading problem that plagued us for a while.

Basically I have trust issues with kube, I'll check to see if I can recreate any of those old issues but v1 was nonexistently documented since it was essentially a laky learning exercise!

I'll look over it and if there's no odd behaviour this is essentially long overdue cleanup ❤️

**Comment by ZZZank** [comment]
Another problem is the use of annotation in `fr_fr.json`. JSON does not support annotation, so I change it. Now the annotation will cause a warn, but not an error. Examples below.
### Before

`    ///// NOM DES OBJETS ICI /////`
### After

`    "":"///// NOM DES OBJETS ICI /////",`

**Comment by ZZZank** [comment]
Oops, there's a problem. 
I created a custom texture for ae2::printed_calculation_processor, and load the texture through resources pack, and the result should be:
image

but the reality is:
image
The texture did get overwrittened by KubeJS, but not by custom resource pack. 
It seems that KubeJS is loading resources using a hidden res-pack, whose priority is `higher` than those res-packs added manually. 
Maybe we need to make assets loaded `only through load-my-resources`, so that players can use resource packs to change textures that are already modified.

**Comment by Laskyyy** [comment]
OH! You remind me of why I did this in the first place now, Jamiscus over on the discord wanted to make resource packs for overwriting some of our textures to match vanilla copper palletes, custom tweaks etc... But this was impossible when loaded through kube, I believe *that* was the reason I moved it to resources instead.

Aye, so I think loading through just the resource mod, or keeping lang files separated are the choices it seems.

**Comment by ZZZank** [comment]
That's a problem, so I think I might have add load-my-resources back, and move these assets to `resources/`, what do you think?
But at the same time, I think there's no need to seperate lang files, because 
a) there are resources packs that will change `lang.json`, like [this](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package) .
b) translators can get all lang files easily by simply searching `en_us.json`. 
c) gathering resources up in one place is good for management, and prevents confusion of having 2 res-path. 

So I think adding load-my-resources back, and move the whole asset folder is a solution, does that work for you? :)

**Comment by Laskyyy** [comment]
I agree that's likely going to be the simplest choice. Perhaps if wanting to make it clearer that the 'resources' folder is all the textures and lang files it'd be possible to change the folder name / location in their config to something that makes more sense. Unsure how necessary that would be, as most the people who touch that folder at this point already know where and what it is by now.

`##[general]

[The path to load resources from.]
S:resource_path = 'resources/';`

**Comment by ZZZank** [comment]
I think this PR is good to go now, my internet struggles in the last 30 mins, so :(

not as ideal as I would expect, but it is a cleanup.
